File: system-test/dotnet/Driver.Adapter/Shared/Client/Http/HttpStatus.cs
namespace Driver.Adapter.Shared.Client.Http;

public static class HttpStatus
{
    public const int Ok = 200;
    public const int Created = 201;
    public const int Accepted = 202;
    public const int NoContent = 204;
    public const int BadRequest = 400;
    public const int Unauthorized = 401;
    public const int Forbidden = 403;
    public const int NotFound = 404;
    public const int Conflict = 409;
    public const int InternalServerError = 500;
    public const int BadGateway = 502;
    public const int ServiceUnavailable = 503;
}

File: system-test/java/src/main/java/com/optivem/shop/testkit/driver/adapter/shared/client/http/HttpStatus.java
package com.optivem.shop.testkit.driver.adapter.shared.client.http;

public class HttpStatus {
    public static final int OK = 200;
    public static final int CREATED = 201;
    public static final int ACCEPTED = 202;
    public static final int NO_CONTENT = 204;
    public static final int BAD_REQUEST = 400;
    public static final int UNAUTHORIZED = 401;
    public static final int FORBIDDEN = 403;
    public static final int NOT_FOUND = 404;
    public static final int CONFLICT = 409;
    public static final int INTERNAL_SERVER_ERROR = 500;
    public static final int BAD_GATEWAY = 502;
    public static final int SERVICE_UNAVAILABLE = 503;

    private HttpStatus() {
        // Utility class - no instantiation
    }
}


